pkg/injector/webhook.go

```diff
@@ -332,6 +360,27 @@ func isAnnotatedForInjection(annotations map[string]string, objectKind string, o
 	return
 }
 
+func isAnnotatedForOutboundPortExclusion(annotations map[string]string, objectKind string, objectName string) (exists bool, ports []string, err error) {
```

Comment:
nit: let's drop the naked return when too many args are being returned. Returning explicitly is a bit clearer.